tools/benchmark/cmd/watch.go

```diff
@@ -163,14 +170,31 @@ func doWatch(stream etcdserverpb.Watch_WatchClient, requests <-chan etcdserverpb
 		results <- result{errStr: errStr, duration: time.Since(st)}
 		bar.Increment()
 	}
+	wg.Done()
+
+	<-putStartNotifier
+
 	for {
+		st := time.Now()
 		_, err := stream.Recv()
 		var errStr string
 		if err != nil {
 			errStr = err.Error()
 		}
-		results <- result{errStr: errStr}
-		bar.Increment()
```

Comment:
if we are measuring the watch receiving rate, we should increase the bar here instead of the in the put routine.